### DISCUSSION TITLE
Webhooks not triggering

### DISCUSSION THREAD

**Discussion by exiz01** [netbox-community/netbox#8753] [Q&A] [answered]
### NetBox version

v2.11.7

### Python version

3.8

### Steps to Reproduce

1. Clean install of netbox 2.11.7
2. Add any webhook

### Expected Behavior

Jobs should be queued but nothing happens

### Observed Behavior

Webhooks are not triggering, there are no jobs on RQ queues (https://netbox.transtelco.net/admin/background-tasks/)
Jobs were working fine but they stoped working abruptly

**Comment by candlerb** [comment, marked as answer]
Logs would be your first port of call.  Anything logged by netbox itself?  `journalctl -eu netbox`

Is redis up and running and reachable by Netbox?

**Reply by exiz01** [reply]
Redis is running and reachable
Logs show no errors, after restart:

> Feb 28 05:56:07 netbox systemd[1]: Stopping NetBox WSGI Service...
> Feb 28 05:56:07 netbox gunicorn[1732761]: [2022-02-28 05:56:07 +0000] [1732761] [INFO] Handling signal: term
> Feb 28 05:56:07 netbox gunicorn[1732779]: [2022-02-28 05:56:07 +0000] [1732779] [INFO] Worker exiting (pid: 1732779)
> Feb 28 05:56:08 netbox gunicorn[1732780]: [2022-02-28 05:56:08 +0000] [1732780] [INFO] Worker exiting (pid: 1732780)
> Feb 28 05:56:08 netbox gunicorn[1732777]: [2022-02-28 05:56:08 +0000] [1732777] [INFO] Worker exiting (pid: 1732777)
> Feb 28 05:56:24 netbox gunicorn[1732776]: [2022-02-28 05:56:24 +0000] [1732776] [INFO] Worker exiting (pid: 1732776)
> Feb 28 05:56:30 netbox gunicorn[1732778]: [2022-02-28 05:56:30 +0000] [1732778] [INFO] Worker exiting (pid: 1732778)
> Feb 28 05:56:31 netbox gunicorn[1732761]: [2022-02-28 05:56:31 +0000] [1732761] [INFO] Shutting down: Master
> Feb 28 05:56:31 netbox systemd[1]: netbox.service: Succeeded.
> Feb 28 05:56:31 netbox systemd[1]: Stopped NetBox WSGI Service.
> Feb 28 05:56:31 netbox systemd[1]: Started NetBox WSGI Service.
> Feb 28 05:56:31 netbox gunicorn[1732886]: [2022-02-28 05:56:31 +0000] [1732886] [INFO] Starting gunicorn 20.1.0
> Feb 28 05:56:31 netbox gunicorn[1732886]: [2022-02-28 05:56:31 +0000] [1732886] [INFO] Listening at: http://127.0.0.1:8001 (1732886)
> Feb 28 05:56:31 netbox gunicorn[1732886]: [2022-02-28 05:56:31 +0000] [1732886] [INFO] Using worker: gthread
> Feb 28 05:56:31 netbox gunicorn[1732900]: [2022-02-28 05:56:31 +0000] [1732900] [INFO] Booting worker with pid: 1732900
> Feb 28 05:56:31 netbox gunicorn[1732901]: [2022-02-28 05:56:31 +0000] [1732901] [INFO] Booting worker with pid: 1732901
> Feb 28 05:56:31 netbox gunicorn[1732902]: [2022-02-28 05:56:31 +0000] [1732902] [INFO] Booting worker with pid: 1732902
> Feb 28 05:56:31 netbox gunicorn[1732903]: [2022-02-28 05:56:31 +0000] [1732903] [INFO] Booting worker with pid: 1732903
> Feb 28 05:56:31 netbox gunicorn[1732904]: [2022-02-28 05:56:31 +0000] [1732904] [INFO] Booting worker with pid: 1732904


Edit: I just realize only "deleted" events are triggering

**Reply by candlerb** [reply]
Just to rule out the obvious, you haven't disabled triggering on other event types?

This is what it looks like in Netbox v3.1.8:

![image](https://user-images.githubusercontent.com/44789/156148927-c5507552-9419-4a7e-960c-966e61d52e87.png)

I don't have an instance of v2.11 any more to show what it looks like there, but I think it was in the Django "Admin" section.

**Reply by exiz01** [reply]
Hi, yes, all webhooks are enabled, anyways I made a clean install of version 2.11.8 (previously 2.11.7) keeping the database and everything seems to be working fine now, I'm thinking someone moved something on my previous installation.
Planning to upgrade to 3.2 once a stable version is released.

Thank you for your help.

**Reply by candlerb** [reply]
No problem.

It's worth going via 3.1, since there are some things which are deprecated in 3.0/3.1 but will be removed in 3.2; running 3.1 gives you a chance to move data to the new models before the fields are deleted. (Site contacts and Site ASNs in particular).
